hubblestack/files/hubblestack_nova/win_secedit.py

```diff
@@ -384,6 +384,12 @@ def _account_audit(current, __sidaccounts__):
     ret_string = ''
     if __sidaccounts__:
         for usr in user_list:
+            if usr.startswith("*S-"):
+                if not ret_string:
+                    ret_string = current
+                else:
+                    ret_string = current
```

Comment:
This if-else block does the same thing in both branches. If that's intentional, then it can be removed. If it's unintentional it's a bug.

Reply:
I was trying to be smart with my yaml.  In the end you are right, it didn't makes sense and I just fixed my yaml.